turbopack/crates/turbopack-tests/tests/snapshot/imports/resolve_error_esm/issues/__l___Module not found____c__ Can't resolve __c_'d-8707b3.txt

```diff
@@ -14,4 +14,6 @@ error - [resolve] [project]/turbopack/crates/turbopack-tests/tests/snapshot/impo
   | Parsed request as written in source code: module "does-not-exist" with subpath '/path'
   | Path where resolving has started: [project]/turbopack/crates/turbopack-tests/tests/snapshot/imports/resolve_error_esm/input/index.js
   | Type of request: EcmaScript Modules request
-  |
\ No newline at end of file
+  | 
+  Import trace for requested module:
+    [project]/turbopack/crates/turbopack-tests/tests/snapshot/imports/resolve_error_esm/input/index.js [test] (ecmascript)
```

Comment:
This is probably an edge case, but maybe filter out the trace if it has length 1 (i.e. doesn't provide any additional info?)